CONTRIBUTING.rst

```diff
@@ -304,6 +305,22 @@ Debugging Tips
      $ npm install
      $ node_modules/.bin/grunt build # -> dest/*.global.js
 
+
+Branch Model
+------------
+
+Sphinx project uses following branches for developing.
+
+``master``
+    Used for main development.  All improvement and refactoring, bugfixes are allowed.
+
+``X.Y`` (cf. 1.7)
+    Used to maintain current stable release.  Only bugfixes and stable changes are allowed.
+    Basically, latest one major version is activated.
```

Comment:
Only the most recent stable release is currently retained. When a new version is released, the old release branch will be deleted and replaced by an equivalent tag.